### PR TITLE
FusionInventory: 2.3.21 -> 2.6

### PR DESCRIPTION
**Pull request by spacefrogg**
###### Motivation for this change
Update and fix for 20.09

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/monitoring/fusion-inventory/default.nix [hunk 1]
```diff
@@ -68,6 +54,8 @@ perlPackages.buildPerlPackage rec {
     XMLTreePP
   ];
 
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/servers/monitoring/fusion-inventory/default.nix:57]: Can you add a comment why the tests are disabled?

Comment (spacefrogg) [reply pkgs/servers/monitoring/fusion-inventory/default.nix:58]: Done.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102034` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>fusionInventory</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102034` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>fusionInventory</li>
  </ul>
</details>

File: pkgs/servers/monitoring/fusion-inventory/default.nix [hunk 2]
```diff
@@ -4,29 +4,15 @@
...
     repo = "fusioninventory-agent";
     rev = version;
-    sha256 = "034clffcn0agx85macjgml4lyhvvck7idn94pqd2c77pk6crvw2y";
+    sha256 = "12grxf56kha71p9rd4p6kfp247c91skv5l82ckjyis7lz3m2dsqc";
   };
 
-  patches = [
-    ./remove_software_test.patch
-    # support for os-release file
-    (fetchurl {
```
Comment (SuperSandro2000) [inline pkgs/servers/monitoring/fusion-inventory/default.nix:19]: Remove fetchurl from inputs.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/servers/monitoring/fusion-inventory/default.nix (hunk 2 above)
Comment (spacefrogg) [reply pkgs/servers/monitoring/fusion-inventory/default.nix:19]: But the _are_ removed. I removed the complete `patches = [ ... ];` attribute. You were commenting on the removed part of the patch. ;)

Comment (SuperSandro2000) [reply pkgs/servers/monitoring/fusion-inventory/default.nix:19]: I mean the the fetchurl in line 1.

Comment (spacefrogg) [reply pkgs/servers/monitoring/fusion-inventory/default.nix:19]: Done.

**Comment by spacefrogg** [comment]
Updated to the 2.6 release.

**Comment by spacefrogg** [comment]
Poke.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102034` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>fusionInventory</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102034` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>fusionInventory</li>
  </ul>
</details>

**Pull request merged**